docroot/index.php

```diff
@@ -4,6 +4,8 @@
 define('CUR_PATH', dirname(__FILE__) . DIRECTORY_SEPARATOR);
 //define('TPL_PATH', CUR_PATH . 'templates' . DIRECTORY_SEPARATOR);
 
+$title_page = 'Home'

```

Comment:
yeah, i caught that way earlier. check latest commit.

september 2014. :)

https://github.com/php/web-windows/pull/3/commits/6463127d10e6357f256f1042325f43e7e0da3107

looks like you figured it out, and merged already though. 👍